webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpacesToolbar.vue

```diff
@@ -9,7 +9,7 @@
       <v-btn
         v-else
         id="addNewSpaceButton"
-        class="btn btn-primary px-0 addNewSpaceButton"
+        class="btn btn-primary px-0 mx-2 mx-sm-0 addNewSpaceButton"
         :height="isMobile && '40' || '40'"
```

Comment:
Are you sure that we need to define a height for a `v-btn` ? I think, it would be better to use `small`, `icon`, `x-small` of `v-btn` properties

Besides, it's the same value for mobile & desktop.

```suggestion
        height="40"
```